src/tests/ftest/soak/soak.yaml

```diff
@@ -100,31 +108,31 @@ jobsmoke:
 # 1, 16 and 32 tasks per node
 job_ior_daos:
     name: job_ior_daos
-    time: 10
+    time: 20
     tasks:
         - 1
         - 16
-        - 32
+        #- 32
     jobspec:
         - ior_daos
 # IOR MPIIO job params; multiple jobs, one job per node;
 # 1, 16 and 32 tasks per node
 job_ior_mpiio:
     name: job_ior_mpiio
-    time: 10
+    time: 20
     tasks:
         - 1
         - 16
-        - 32
+        #- 32
```

Comment:
Same as above

Reply:
I need to open a ticket for this. I can not get 32 task per node to run

Reply:
ditto